src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -616,7 +658,9 @@ Result TimestampOrderingTransactionManager::CommitTransaction(
   // generate transaction id.
   cid_t end_commit_id = current_txn->GetBeginCommitId();
 
-  auto &rw_set = current_txn->GetRWSet();
+  auto &rw_set = current_txn->GetReadWriteSet();
+
+  auto gc_set = current_txn->GetGCSetRef();
```

Comment:
I think the _GetGCSetRef()_ method should be renamed to _GetGCSetPtr()_ to make it less confusing.